FlashcardAPIs/FlashcardAPIs/Controllers/FlashCardCategoriesController.cs

```diff
@@ -54,8 +54,11 @@ public IActionResult Post([FromBody]FlashCardCategory category)
 
         // PUT api/values/5
         [HttpPut("{id}")]
-        public void Put(int id, [FromBody]string value)
+        public IActionResult Put(string id, [FromBody]FlashCardCategory category)
         {
+            var category1 = _mongoDbWriteRepository.Get<FlashCardCategory>(category.Id);
```

Comment:
Please use suitable name for variable. You can use responseCategory instead of category1